util/pkg/vfs/context.go

```diff
@@ -79,6 +79,9 @@ func (c *VFSContext) ReadFile(location string) ([]byte, error) {
 			case "aws":
 				httpURL := "http://169.254.169.254/latest/" + u.Path
 				return c.readHttpLocation(httpURL, nil)
+			case "digitalocean":
+				httpURL := "http://169.254.169.254/metadata/v1" + u.Path
```

Comment:
Oops - looks like you're right on the trailing slash not being required (i.e. aws should be `http://169.254.169.254/latest` not `http://169.254.169.254/latest/`)  Thanks for noticing 👍